style.css

```diff
@@ -0,0 +1,187 @@
+.mobile-header {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
> * [ ]  Please try to stick with the design as much as possible to make your design looks good and it can help to create a more familiar, consistent, and intuitive user experience, which can increase user engagement and satisfaction with the website. Please add more space at left and right-hand side of the header.
> 
> Figma Design	Your Design
> ![Screenshot from 2023-04-29 16-07-29](https://user-images.githubusercontent.com/66476329/235300977-10521b41-b27c-4c3b-be0d-a1f376e4ad74.png)
> 
> ![Screenshot from 2023-04-29 16-31-47](https://user-images.githubusercontent.com/66476329/235300982-c348319d-7581-4879-9859-62e5631dd07a.png)

Please, use a background image here as requested by the previous reviewer.